test/functional/Progress.hs

```diff
@@ -37,7 +37,7 @@ tests =
 
               (codeLensResponse, activeProgressTokens) <- expectProgressMessagesTill
                 (responseForId SMethod_TextDocumentCodeLens lspId)
-                ["Setting up testdata (for T1.hs)", "Processing", "Indexing"]
+                ["Setting up testdata (for T1.hs)", "Processing"]
```

Comment:
Removing this Indexing because it was making the test flaky (sometimes appeared _after_ the code lens reponse).
As suggested by @fendor the only Indexing message that happens is related to files not related to this test, so it also semantically doesn't belong here.